editor/src/components/canvas/canvas-strategies/strategies/reparent-helpers/reparent-strategy-parent-lookup.ts

```diff
@@ -55,11 +55,28 @@ export function getReparentTargetUnified(
     allowSmallerParent,
   )
 
+  // with current parent under cursor filter ancestors from reparent targets
+  const currentParentUnderCursor =
+    reparentSubjects.type === 'EXISTING_ELEMENTS'
+      ? validTargetparentsUnderPoint.find((targetParent) =>
+          EP.isParentOf(targetParent, reparentSubjects.elements[0]),
+        ) ?? null
+      : null
```

Comment:
It looks like this would make more sense as a boolean since we do nothing with this value other than check that it exists

Reply:
it is used in line 68